### PR TITLE
docs(cli): update grammar and spelling mistakes in readme.md

### PR DESCRIPTION
**Pull request by prasanthlouis**
Updating some grammar and spelling mistakes.

----

### All Submissions:

* [x] Have you followed the guidelines in our [Contributing guide?](https://github.com/aws/aws-cdk/blob/main/CONTRIBUTING.md)

### Adding new Unconventional Dependencies:

* [ ] This PR adds new unconventional dependencies following the process described [here](https://github.com/aws/aws-cdk/blob/main/CONTRIBUTING.md/#adding-new-unconventional-dependencies)

### New Features

* [ ] Have you added the new feature to an [integration test](https://github.com/aws/aws-cdk/blob/main/INTEGRATION_TESTS.md)?
	* [ ] Did you use `yarn integ` to deploy the infrastructure and generate the snapshot (i.e. `yarn integ` without `--dry-run`)?

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: packages/aws-cdk/README.md [hunk 1]
```diff
@@ -336,19 +336,19 @@ The `progress` key can also be specified as a user setting (`~/.cdk.json`)
 
 #### CloudFormation Change Sets vs direct stack updates
 
-By default CDK will create a CloudFormation change with the changes that will
-be deployed, and then executes it. This behavior can be controlled with the
+By default, CDK will create a CloudFormation change with the changes that will
```
Comment (TheRealAmazonKendra) [inline packages/aws-cdk/README.md:339]:
```suggestion
By default, CDK creates a CloudFormation change set with the changes that will
```

**Review by TheRealAmazonKendra — CHANGES_REQUESTED** [review]
Thank you for your contribution! Please see the PR Linter feedback and mine inline.

File: packages/aws-cdk/README.md (hunk 1 above)
Comment (prasanthlouis) [reply packages/aws-cdk/README.md]: Thank you so much!

**Comment by prasanthlouis** [comment]
> Thank you for your contribution! Please see the PR Linter feedback and mine inline.

Thank you so much!

**Review by TheRealAmazonKendra — APPROVED** [review]

**Pull request merged**